e2e-tests/authd-msentraid/common/browser_login.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+import gi, os, sys
+
+gi.require_version("Gtk", "3.0")
+gi.require_version("Gdk", "3.0")
+gi.require_version("WebKit2", "4.1")
+
+from gi.repository import Gdk, Gtk  # type: ignore # The interpreter usually does not recognize imports from gi.repository
+
+# The import will be resolved at runtime, which means that the directory structure will be something like:
+# test_run_dir/
+#   broker-common/
+#       browser_login.py  <- this file
+#   authd-common/
+#      BrowserWindow.py <- the dependency
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "../authd-common")))
+from BrowserWindow import BrowserWindow, ascii_string_to_key_events  # type: ignore # This is resolved at runtime
+
+
+if __name__ == "__main__":
+    if os.getenv("RUN_ONSCREEN") is None and "RUNNING_OFFSCREEN" not in os.environ:
+        os.execv(
+            "/usr/bin/env",
+            [
+                "/usr/bin/env",
+                "RUNNING_OFFSCREEN=1",
+                "GDK_BACKEND=x11",
+                "xvfb-run",
+                "-a",
+                sys.executable,
+            ]
+            + sys.argv,
+        )
+
+    if len(sys.argv) < 4:
+        print("Usage: BrowserWindow.py <username> <password> <code>")
+        sys.exit(1)
+
+    username = sys.argv[1]
+    password = sys.argv[2]
+    device_code = sys.argv[3]
```

Comment:
Not a big deal, but I feel you can have this with 4 lines of [`argparse`](https://docs.python.org/3/library/argparse.html)

Reply:
I thought about it, but it's not worth adding the dependency for something as simple as this, especially for a script that is not intended to be run by people.

Reply:
FYI `argparse` is part of the standard library